### ISSUE TITLE
Better HELP items

### ISSUE THREAD

**Issue by Geert-Jan-DH** [frankframework/frankframework#5738]
As a user I've wondered a lot why there's no user HELP (in an intuitive place) in the menu.

The HELP I see is al listing of pipes(?) and / or adepters and other stuff I don't want to know,  with every variable explaned what doesn't help me figuring out how the FF is working.

How FF works in general, here to find the options to set in configs and Helm or any other instruction for Ladybug or something else is very hard to find. It might be there, but I didn't find it over the last year.

Suggestion: add https://frank-manual.readthedocs.io/en/latest/ as the first (or general) help-page to the FF.

**Comment by nielsm5** [comment]
You might be mixing up the 'technical' [element documentation](https://frankdoc.frankframework.org/) with the [user manual](https://frank-manual.readthedocs.io/).

**Comment by checkiecheck** [comment]
Would suggest to restructure several info links into one menu item on the right
<img width="314" alt="image" src="https://github.com/ibissource/iaf/assets/79155146/9d38165b-2819-4c09-ac82-df1fc7056212">
these are from the 'information' popup, which could better reside in the suggeste menu item, together with the frank doc link.

**Comment by Matthbo** [comment]
Would something like this be clearer?
![image](https://github.com/frankframework/frankframework/assets/4380412/18c1f2ea-51eb-469a-b2b7-9e18caa4956c)

**Comment by Geert-Jan-DH** [comment]
Yes! And change the order..? People who know what to look for (the Doc) don't mind. The people looking for help will pick the first one they see 🤔


Geert-Jan Pes

Functioneel beheerder senior, Informatievoorziening / Team vergunningen

06 52 51 32 63


Gemeente Den Haag
Dienst Bedrijfsvoering Informatisering & Automatisering (DBV I&A)

Spui 70
2511 BT Den Haag
www.denhaag.nl<http://www.denhaag.nl>

*** verstuurd vanaf mijn smartphone en daarom misschien wat beperkt ***
________________________________

**Comment by nielsm5** [comment]
This has since been implemented.
